nav2_mppi_controller/include/nav2_mppi_controller/tools/utils.hpp

```diff
@@ -612,6 +606,80 @@ struct Pose2D
   float x, y, theta;
 };
 
+/**
+ * @brief Shift the columns of a 2D Eigen Array or scalar values of
+ *    1D Eigen Array by 1 place.
+ * @param e Eigen Array
+ * @param direction direction in which Array will be shifted.
+ *     1 for shift in right direction and -1 for left direction.
```

Comment:
This appears to only work with `1` and `-1`, so should we error check by throwing if a non-`1` or `-1` is given as an input? I could see someone making the mistake of trying to roll by 2+ in some direction and being surprised that it didn't work